tests/translator/input/error_resource_properties_not_dict.yaml

```diff
@@ -0,0 +1,4 @@
+Resources:
+  Function:
+    Type: AWS::Serverless::Function
```

Comment:
Do we have a test where `Type` doesn't exist? I saw that in a log dive as well.

Reply:
Yes I believe this is already covered.